types/karma-jasmine/karma-jasmine-tests.ts

```diff
@@ -18,12 +18,12 @@ const testOne = (config: karma.Config) => {
 const testTwo = (config: karma.Config) => {
     config.set({
         client: {
-            // $ExpectType { random: true; seed: number; oneFailurePerSpec: true; failFast: true; timeoutInterval: number; }
+            // $ExpectType { random: true; seed: number; stopSpecOnExpectationFailure: true; stopOnSpecFailure: true; timeoutInterval: number; }
             jasmine: {
                 random: true,
                 seed: 4321,
-                oneFailurePerSpec: true,
-                failFast: true,
+                stopSpecOnExpectationFailure: true,
```

Comment:
karma-jasmine just passes the provided config through to jasmine-core, so although the configuration changes affect karma-jasmine's users, they shouldn't affect karma-jasmine itself.